tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -221,6 +259,11 @@ Status FilterPipeline::filter_chunks_forward(
           &output_metadata,
           &output_data));
 
+      // Final tile type will be the output type of last filter in pipeline.
```

Comment:
Can't we do this in the writer when constructing the tile? I'd like to avoid changing the type after the tile was created.